stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/endpoint/ChannelApi.kt

```diff
@@ -14,10 +14,11 @@
  * limitations under the License.
  */
 
-package io.getstream.chat.android.client.api2
+package io.getstream.chat.android.client.api2.endpoint
```

Comment:
I'm curious, was the wrong package name a part of the reason why shaking the tree made this part of the SDK fall out?

Maybe we should lint this more aggressively or even introduce it as a Detekt rule. @JcMinarro